app/styles/player.styl

```diff
@@ -0,0 +1,130 @@
+div[role='player']
```

Comment:
`role` `player` dosn't exists (https://www.w3.org/TR/wai-aria/roles). You can use a _class_ or an _id_ if you're sure to have only one instance in your app markup at one time.